static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -572,6 +679,7 @@ export const bids: (string, PrebidSize[]) => PrebidBid[] = (
                 bid.labelAll = bidder.labelAll;
             }
         }
+        console.log(`Bid for ${slotId} ${bidder.name}: ${JSON.stringify(bid)}`);
```

Comment:
Does enabling the the prebid debugging param show enough information? Do we need the console.log here? 🙏

Reply:
Oops, accidentally left in!